lib/bundler/cli/config.rb

```diff
@@ -59,7 +59,7 @@ def run
               "This will take precedence over the global value you are setting"
           end
 
-          if global = locations[:global]
+          if global = locations[:global] and global != locations[:global]
```

Comment:
I don't think we have a formal bundler style guide, but I personally follow CocoaPods', which is a modified version of bbatsov's